src/shared/ui/Message/MessagesWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { memo, useEffect } from 'react';
+import { Stack } from '@mui/material';
+import { getMessages } from 'entities/Task';
+import { Message } from './Message';
+import { useAppSelector } from '../../lib/hooks/redux';
+import { getPermission } from '../../lib/notification';
+
+import { MessageStyle as styles } from './Message.style';
+
+export const MessagesWrapper = memo(() => {
+    const messages = useAppSelector(getMessages);
+
+    useEffect(() => {
+        getPermission();
+    }, []);
+
+    if (messages.length) {
+        return (
+            <Stack sx={styles.wrapper} spacing={1}>
+                {messages.map((message) => <Message message={message} key={message.taskID} />)}
```

Comment:
you shouldn't use only `message.taskID` as a `key` when it's possible to display several messages for a single task

Reply:
Changed it to a random ID for each message

Reply:
I would suggest something like this `key={`${message.taskID}-${message}`}`